SF iOS/SF iOS/Location/TravelTime/TravelTimeService.m

```diff
@@ -93,12 +93,14 @@ - (AsyncBlockOperation *)travelTimeCalculationWithRequest:(MKDirectionsRequest *
     return [[AsyncBlockOperation alloc] initWithAsyncBlock:^(dispatch_block_t  _Nonnull completionHandler) {
         MKDirections *direction = [[MKDirections alloc] initWithRequest:request];
         [direction calculateETAWithCompletionHandler:^(MKETAResponse * _Nullable response, NSError * _Nullable error) {
+            NSTimeInterval travelTime;
             if (!response) {
                 NSLog(@"Could not get travel time for request:\n%@\n%@", request, error);
-                resultHandler(nil);
-                completionHandler();
```

Comment:
So we just like, weren't doing anything huh?

Reply:
Nope.

Reply:
Depending on your perspective, either it wasn't doing anything (to handle the no-response case) or it was doing too much (running the handlers twice). It's a ~~Choose Your Own Adventure™~~ piece of interactive fiction!